### PR TITLE
W-11593638-permission-model-for-sharing-fa

### PR DESCRIPTION
**Pull request by failup**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/permissions-by-product.adoc [hunk 1]
```diff
@@ -52,8 +52,11 @@ Or:
 
 == Design Center
 
+* Design Center Viewer
+* Design Center Creator
 * Design Center Developer
 
+
```
Comment (lbauret) [inline modules/ROOT/pages/permissions-by-product.adoc]: We may need to have a similar division as Exchange below, with:

Organization level permissions: 
* Design Center Viewer
* Design Center Creator
* Design Center Developer

Project level permissions:
* Project administrator
* Project editor
* Project viewer

**Review by iamerico — DISMISSED** [review]
LGTM

File: modules/ROOT/pages/permissions-by-product.adoc (hunk 1 above)
Comment (ankitamule) [reply modules/ROOT/pages/permissions-by-product.adoc]: Agree with Leandro here! We should call out both org level and project level permissions! Thank you

**Review by lbauret — APPROVED** [review]
LGTM!

**Review by iamerico — APPROVED** [review]
LGTM

**Review by kevintroller — APPROVED** [review]

**Pull request merged**